src/model/utils.ts

```diff
@@ -81,3 +81,7 @@ export function getCplaceAscNodeModulesBinPath(): string {
 export function getProjectNodeModulesBinPath(): string {
     return resolve(getProjectNodeModulesPath(), '.bin');
 }
+
+export function isArtifactsOnlyBuild(): boolean {
+    return process.env.CPLACE_BUILD_WITHOUT_PARENT_REPOS === 'true';
```

Comment:
regarding local development: as a dev I'd prefer to be able to set this additionally with a flag when calling cplace-asc from terminal. Ideally, the same flag would be provided by the cplace-gradle-plugin for calling associated tasks.